config/config.go

```diff
@@ -125,6 +125,8 @@ type Coordinator struct {
 	// to the PurgeByExternalDelete function of the l2db which deletes
 	// pending txs externally marked by the column `external_delete`
 	PurgeByExtDelInterval Duration `validate:"required" env:"HEZNODE_COORDINATOR_PURGEBYEXTDELINTERVAL"`
+	// ProverWaitReadTimeout
+	ProverWaitReadTimeout Duration `env:"HEZNODE_COORDINATOR_PROVERWAITREADTIMEOUT"`
```

Comment:
Seems that this new parameter is required.
```suggestion
	ProverWaitReadTimeout Duration `validate:"required" env:"HEZNODE_COORDINATOR_PROVERWAITREADTIMEOUT"`
```

Reply:
@ARR552 thank you!